packages/interaction-kit/src/commands/options/attachment.ts

```diff
@@ -0,0 +1,13 @@
+import { ApplicationCommandOptionType } from "discord-api-types/v9";
+import Option, { BaseOptionArgs } from "./option";
+
+export default class AttachmentOption extends Option {
```

Comment:
w

Reply:
Is this just duplicating what builders has? 👀